includes/class-llms-blocks-assets.php

```diff
@@ -176,20 +177,21 @@ public function editor_assets() {
 	 */
 	private function use_bc_assets() {
 		return ( ! LLMS_Forms::instance()->are_requirements_met() &&
-			/**
-			 * Filter allowing opt-out of block editor backwards compatibility scripts.
-			 *
-			 * @since 2.0.0
-			 *
-			 * @example
-			 * ```
-			 * // Disable backwards compatibility scripts.
-			 * add_filter( 'llms_blocks_load_bc_scripts', '__return_false' );
-			 * ```
-			 *
-			 * @param boolean $load_scripts Whether or not to load the scripts.
-			 */
-			apply_filters( 'llms_blocks_load_bc_scripts', true )
+				 /**
+				  * Filter allowing opt-out of block editor backwards compatibility scripts.
+				  *
+				  * @since 2.0.0
+				  *
+				  * @param boolean $load_scripts Whether or not to load the scripts.
+				  *
+				  * @example
+				  *        ```
+				  *        // Disable backwards compatibility scripts.
+				  *        add_filter( 'llms_blocks_load_bc_scripts', '__return_false' );
+				  *        ```
+				  *
+				  */
+				 apply_filters( 'llms_blocks_load_bc_scripts', true )
```

Comment:
This still needs to be resolved.
You inverted some tags position. Can you keep the original order?
```
			/**
			 * Filter allowing opt-out of block editor backwards compatibility scripts.
			 *
			 * @since 2.0.0
			 *
			 * @example
			 * ```
			 * // Disable backwards compatibility scripts.
			 * add_filter( 'llms_blocks_load_bc_scripts', '__return_false' );
			 * ```
			 *
			 * @param boolean $load_scripts Whether or not to load the scripts.
			 */
```
was just fine.